docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: model
+title: Typo Detector
+author: John Snow Labs
+name: distilbert_token_classifier_typo_detector
+date: 2022-01-19
+tags: [typo, distilbert, en, open_source]
+task: Named Entity Recognition
+language: en
+edition: Spark NLP 3.3.4
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model was imported from `Hugging Face` ([link](https://huggingface.co/m3hrdadfi/typo-detector-distilbert-en)) and it's been trained on NeuSpell corpus to detect typos, leveraging `DistilBERT` embeddings and `DistilBertForTokenClassification` for NER purposes. It classifies typo tokens as `PO`.
+
+## Predicted Entities
+
+`PO`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/distilbert_token_classifier_typo_detector_en_3.3.4_3.0_1642581005021.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "en")\
+       .setInputCols(["document"])\
+       .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+      .setInputCols(["sentence"])\
+      .setOutputCol("token")
+
+tokenClassifier = DistilBertForTokenClassification.pretrained("distilbert_token_classifier_typo_detector", "en"))\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+text = """He had also stgruggled with addiction during his tine in Congress."""
+data = spark.createDataFrame([[text]]).toDF("text")
+
+result = nlpPipeline.fit(data).transform(data)
+```
+```scala
+val documentAssembler = DocumentAssembler()
+      .setInputCol("text")
+      .setOutputCol("document")
+
+val sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "en")
+       .setInputCols(Array("document"))
+       .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()
+      .setInputCols(Array("sentence"))
+      .setOutputCol("token")
+
+val tokenClassifier = DistilBertForTokenClassification.pretrained("distilbert_token_classifier_typo_detector", "en"))\
```

Comment:
Two parenthesis, only one needed. And remove also `\` in Scala is not needed as you correctly told me some time ago :)

Reply:
@josejuanmartinez 
it is fixed!